cmd/state-exec/cmd_windows.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"os/exec"
+	"syscall"
+)
+
+func Command(name string, arg ...string) *exec.Cmd {
+	cmd := exec.Command(name, arg...)
+
+	// Go currently does not escape arguments properly on Windows, it account for spaces and tab characters, but not
+	// other characters that need escaping such as `<` and `>`.
+	// This can be dropped once we update to a Go version that fixes this bug: https://github.com/golang/go/issues/68313
+	cmd.SysProcAttr = &syscall.SysProcAttr{CmdLine: makeCmdLine(cmd.Args)}
+
+	return cmd
+}
```

Comment:
Note this is purposely duplicating logic in osutils, because a design principle behind executors is that we avoid reusing internal libraries so as to keep the executable as lightweight as possible.

Reply:
Yeah, I've accidentally included things that blew up the exe size. Fortunately we have a test to catch it 😅